<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CBasicStreamEditorClass</class>
  <widget class="QWidget" name="CBasicStreamEditorClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>302</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>CBasicStreamEditor</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
        <number>0</number>
      </property>
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item>
        <widget class="QFrame" name="frameHoldupControls">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
            <property name="spacing">
              <number>5</number>
            </property>
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="bottomMargin">
              <number>0</number>
            </property>
            <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                  <widget class="QLabel" name="label">
                    <property name="text">
                      <string>Time points [s]</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="CQtTable" name="timePointsTable">
                    <property name="selectionMode">
                      <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <attribute name="horizontalHeaderVisible">
                      <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderCascadingSectionResizes">
                      <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderStretchLastSection">
                      <bool>true</bool>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                      <bool>false</bool>
                    </attribute>
                    <column>
                      <property name="text">
                        <string>Time</string>
                      </property>
                    </column>
                    <property name="toolTip">
                      <string>List of time points in the selected holdup/stream</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>List of time points in the selected holdup/stream</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="addTimePoint">
                    <property name="text">
                      <string>+</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Add time point for the selected holdup/stream</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Add time point for the selected holdup/stream</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="removeTimePoint">
                    <property name="text">
                      <string>-</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Remove time point for the selected holdup/stream</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Remove time point for the selected holdup/stream</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="removeAllTimePoints">
                    <property name="text">
                      <string>Remove all</string>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Remove all time points for the selected holdup/stream</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Remove all time points for the selected holdup/stream</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QTabWidget" name="mainTabWidget">
                <property name="toolTip">
                  <string>Properties of the selected holdup/stream at selected time point</string>
                </property>
                
                <property name="whatsThis">
                  <string>Properties of the selected holdup/stream at selected time point</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <customwidgets>
    <customwidget>
      <class>CQtTable</class>
      <extends>QTableWidget</extends>
      <header location="global">QtTable.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>timePointsTable</tabstop>
    <tabstop>addTimePoint</tabstop>
    <tabstop>removeTimePoint</tabstop>
    <tabstop>removeAllTimePoints</tabstop>
    <tabstop>mainTabWidget</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>